pkg/spec/gitops.go

```diff
@@ -1014,6 +1014,12 @@ func parseSoftware(top map[string]json.RawMessage, result *GitOps, baseDir strin
 				continue
 			}
 		}
+		if softwarePackageSpec.SHA265 != "" && len(softwarePackageSpec.SHA265) != 64 {
```

Comment:
Transposed digits.

Also, we should regex this, as we want 64 lower-case hex characters, rather than just any 64 chars.

Should also `continue` after this and the below conditions, since if those aren't valid then the rest of the package spec isn't valid.